app/web/features/profile/edit/EditProfile.tsx

```diff
@@ -391,6 +429,7 @@ export default function EditProfileForm() {
               defaultValue={user.aboutMe || DEFAULT_ABOUT_ME_HEADINGS}
               control={control}
               className={classes.field}
+              otherChildren={charsRemainingAboutMe()}
```

Comment:
I'd consider updating this to `children` instead since it's not being used yet, then once you break it up into a separate component, it can read more "naturally" like this:
```tsx
// Intentionally leaving out all of the props for brevity
<ProfileMarkdownInput>
  <IncompleteProfileAlert />
</ProfileMarkdownInput>
```

instead of having to do `otherChildren={...}` (children is a shortcut for allowing more "natural" composition of components for these particular use cases)